packages/api/controllers/operatorServices/operations/deletePropertyEnv.js

```diff
@@ -69,23 +89,35 @@ const deletePropertyEnv = async (req, res, next) => {
     await dataAlias.save();
     await activityStream.insertMany(applicationsActivityStream);
 
-    console.log(deleteAlias);
-    console.log(deleteApplication);
+    log.info(deleteAlias);
+    log.info(deleteApplication);
 
     const deleteCountAlias = await alias.findOne({ propertyName: propertyName, env: env }).remove().exec();
     const deleteCountApplication = await applications.findOne({ propertyName: propertyName, env: env }).remove().exec();
 
-    console.log(deleteCountAlias);
-    console.log(deleteCountApplication);
+    log.info(deleteCountAlias);
+    log.info(deleteCountApplication);
```

Comment:
What is the need for logging this?

Reply:
Okay make the logs with understandable message.

Reply:
Sure rigin

Reply:
@riginoommen I've made the changes

Reply:
You have n't make the changes

Reply:
https://github.com/spaship/spaship/commit/289d0e4a104483ca00b1a72a7080df6fecffb347#diff-8b24b73e2bad5f01ac922751ff09444300ddeefc0bf8b60c9cbc2e9667b08b74R98

Hey could you please check this, probably it's not updated.

Reply:
https://github.com/spaship/spaship/blob/289d0e4a104483ca00b1a72a7080df6fecffb347/packages/api/controllers/operatorServices/operations/deletePropertyEnv.js#L98-L99